### PR TITLE
Dev pushin

### PR DESCRIPTION
**Pull request by Pushin1007**
reload and modified data base on the comments Aleksei Laptev

### CODE REVIEW

**Comment by Pushin1007** [comment]
reload and modified data base on the comments Aleksei Laptev

**Review by DmitrySergienko — APPROVED** [review]
From my side ok!

File: app/src/main/java/com/example/fitnessconstructor/database/Dao.kt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    @Query("SELECT * FROM workout_rest WHERE id IS :id")
+    fun getWorkoutRest(id: Int?): Flow<WorkoutRestEntity>
+
+    // получаем программу стресс теста
+    @Query("SELECT * FROM stress_workout_exercises")
+    fun getStressWorkoutExercises(): Flow<List<StressWorkoutExercisesEntity>>
+
+    // получаем программу программу тренировки в зависимости от её номера и дня
+    @Query("SELECT * FROM workout_exercises WHERE id IS :id AND day IS :day")
+    fun getWorkoutExercises(id: Int?, day: Int?): Flow<WorkoutExercises>
```
Comment (MarDSoul) [inline app/src/main/java/com/example/fitnessconstructor/database/Dao.kt:41]: find by workoutId

**Review by MarDSoul — COMMENTED** [review]
Resolve conflicts please
I think it's better to create StressDao class and split the logic stress and standart workout
don't forget about Hilt - maybe later

**Review by MarDSoul — APPROVED** [review]

**Comment by Pushin1007** [comment]
Let's not split DAO yet
Resolve conflicts the one who merge? I can't. I am the author of the pool

**Pull request merged**